aws/table_aws_backup_recovery_point.go

```diff
@@ -288,3 +297,62 @@ func getAwsBackupRecoveryPoint(ctx context.Context, d *plugin.QueryData, h *plug
 
 	return detail, nil
 }
+
+func getAwsBackupRecoveryPointTags(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	arn := recoveryPointArn(h.Item)
+
+	// Define the regex pattern for the recovery point ARN
+	pattern := `arn:aws:backup:[a-z0-9\-]+:[0-9]{12}:recovery-point:*`
+
+	// Create a regular expression object
+	re := regexp.MustCompile(pattern)
+
+	// Only return the tags associated with the resovery point
+	if !re.MatchString(arn) {
+		return nil, nil
+	}
+
+	// Create Session
+	svc, err := BackupClient(ctx, d)
+	if err != nil {
+		plugin.Logger(ctx).Error("aws_backup_recovery_point.getAwsBackupRecoveryPointTags", "connection_error", err)
+		return nil, err
+	}
+	if svc == nil {
+		// Unsupported region, return no data
+		return nil, nil
+	}
+
+	params := &backup.ListTagsInput{
+		ResourceArn: aws.String(arn),
+	}
+
+	op, err := svc.ListTags(ctx, params)
+	if err != nil {
+
+		var ae smithy.APIError
+		if errors.As(err, &ae) {
+			if ae.ErrorCode() == "ResourceNotFoundException" {
+				return &backup.GetBackupVaultNotificationsOutput{}, nil
```

Comment:
There is some unintentional type error here. This should have nothing to do with backup vault notifications

Reply:
Ahh, My bad updated the return type.